superset-frontend/plugins/plugin-chart-echarts/src/Histogram/controlPanel.tsx

```diff
@@ -0,0 +1,131 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import {
+  GenericDataType,
+  t,
+  validateInteger,
+  validateNonEmpty,
+} from '@superset-ui/core';
+import {
+  ControlPanelConfig,
+  columnChoices,
+  formatSelectOptionsForRange,
+} from '@superset-ui/chart-controls';
+import { showLegendControl, showValueControl } from '../controls';
+
+const config: ControlPanelConfig = {
+  controlPanelSections: [
+    {
+      label: t('Query'),
+      expanded: true,
+      controlSetRows: [
+        [
+          {
+            name: 'column',
+            config: {
+              type: 'SelectControl',
+              label: t('Numeric column'),
+              multi: false,
+              description: t('Select the numeric column to draw the histogram'),
+              default: null,
+              validators: [validateNonEmpty],
+              mapStateToProps: ({ datasource }) => ({
+                choices: columnChoices(datasource, GenericDataType.Numeric),
+              }),
+            },
+          },
+        ],
+        ['groupby'],
+        ['adhoc_filters'],
+        ['row_limit'],
+        [
+          {
+            name: 'bins',
+            config: {
+              type: 'SelectControl',
+              freeForm: true,
+              label: t('Bins'),
+              default: 5,
+              choices: formatSelectOptionsForRange(5, 20, 5),
+              description: t('The number of bins for the histogram'),
+              validators: [validateInteger],
+            },
+          },
+        ],
+        [
+          {
+            name: 'normalize',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Normalize'),
+              description: t('Whether to normalize the histogram'),
```

Comment:
I wonder if such descriptions are valuable - it feels a bit repetitive with the label. Maybe we should either skip the description, or (preferably) describe how normalization works, or provide a link to a resource that explains normalization?

Reply:
That's a good point. I'll see if I can generate some helpful tooltips using GPT.